src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+
+    public Lottery(String color, int number) {
+        super(color, number);
+    }
+
+    public Ball getRundomBall() {
+        int number = new Random().nextInt(100);
```

Comment:
don't forget about making magic numbers like this `100` - a constants, check the other vars a s well